admin-base/core/src/main/java/com/peregrine/admin/servlets/CreateTenantServlet.java

```diff
@@ -158,6 +160,14 @@ protected Response handleRequest(Request request) throws IOException {
             // Create Tenant Group
             String tenantGroupId = toTenant + GROUP_NAME_SUFFIX;
             String tenantUserId = toTenant + USER_NAME_SUFFIX;
+
+            // check if we re trying to create a user that already exists
+            if(isAdmin && userManager.getAuthorizable(tenantUserId) != null) {
+                return new ErrorResponse()
+                .setHttpErrorCode(SC_BAD_REQUEST)
+                .setErrorMessage(FAILED_TO_CREATE_TENANT_SECURITY);
+            }
+            
```

Comment:
we send an error response in case an admin tries to create a site/user pair where the user already exists